src/front-end/src/components/AdminSignup/AdminSignup.js

```diff
@@ -75,7 +81,6 @@ class AdminSignup extends Component {
               label="Name"
               id="nameInput"
               name="nameInput"
-              // must have an input:
```

Comment:
The 'required' field seemed fairly self-explanatory to me, so I removed most of these comments. Let me know if you disagree, and I can add them back in.

Reply:
I agree. Overcommenting is bad.